packages/@rnw-scripts/just-task/just-task.js

```diff
@@ -35,10 +38,32 @@ task('build', series('ts'));
 
 task('rebuild', series('clean', 'build'));
 
+task('depcheck', async () => {
+  const depcheckConfig = path.join(process.cwd(), 'depcheck.config.js');
+  const userDepcheckOptions = fs.existsSync(depcheckConfig) ? require(depcheckConfig) : {};
+  const depcheckOptions = {
+    ...userDepcheckOptions,
+    specials: [
+      depcheck.special.eslint,
```

Comment:
re: @vue comment This is where webpacks plugin model would make sense rather than 'ship the world' :)